packages/cli/src/webhooks/webhook.types.ts

```diff
@@ -26,12 +26,12 @@ export interface IWebhookManager {
 		httpMethod: IHttpRequestMethods,
 	) => Promise<WebhookAccessControlOptions | undefined>;
 
-	executeWebhook(req: WebhookRequest, res: Response): Promise<IWebhookResponseCallbackData>;
+	executeWebhook(req: WebhookRequest, res: Response): Promise<void>;
 }
 
-export interface IWebhookResponseCallbackData {
-	data?: IDataObject | IDataObject[];
-	headers?: object;
-	noWebhookResponse?: boolean;
-	responseCode?: number;
-}
+export type IWebhookResponsePromiseData =
+	| { noWebhookResponse: true }
+	| {
+			noWebhookResponse?: false;
+			response: IN8nHttpFullResponse;
+	  };
```

Comment:
Would it be possible for this to be `type IWebhookResponsePromiseData = null | IN8nHttpFullResponse`?